src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -58,6 +58,10 @@ error.incorrect.length=\
         incorrect length while processing: {0}
 error.create.tempfile=\
         Could not create a temporary file
+error.extract.multiple.dest.dir=\
+        You may not specify more than one directory for extracting the jar
```

Comment:
Perhaps something similar to:

You may not specify  the  '-C' or '--dir' option more than once with the '-x' option

Reply:
Done